doc/man7/migration_guide.pod

```diff
@@ -0,0 +1,1807 @@
+
+See L</Deprecated low-level key object getters and setters>
+
+=item EVP_PKEY_set1_tls_encodedpoint() and EVP_PKEY_get1_tls_encodedpoint().
+
+These functions were previously used by libssl to set or get an encoded public
+key into/from an EVP_PKEY object. With OpenSSL 3.0 these are replaced by the more
+generic functions L<EVP_PKEY_set1_encoded_public_key(3)> and
+L<EVP_PKEY_get1_encoded_public_key(3)>.
+The old versions have been converted to deprecated macros that just call the
+new functions.
+
+=item EVP_PKEY_set1_engine and EVP_PKEY_get0_engine
+
+See L</Providers are a replacement for engines and low-level method overrides>.
+
+=item EVP_PKEY_set_alias_type
+
+This function has been removed. There is no replacement.
+See L</EVP_PKEY_set_alias_type() method has been removed>
+
+=item HMAC_Init_ex, HMAC_Update, HMAC_Final, HMAC_size
+
+See L</Deprecated low-level MAC functions>.
+
+=item HMAC_CTX_new, HMAC_CTX_free, HMAC_CTX_copy, HMAC_CTX_reset,
+HMAC_CTX_set_flags and HMAC_CTX_get_md
+
+See L</Deprecated low-level MAC functions>.
+
+=item i2d_DHparams, i2d_DHxparams
+
+See L</Deprecated low-level key reading and writing functions>
+and L<d2i_RSAPrivateKey(3)/Migration> 
+
+=item i2d_DSAparams, i2d_DSAPrivateKey, i2d_DSAPrivateKey_bio,
+i2d_DSAPrivateKey_fp, i2d_DSA_PUBKEY, i2d_DSA_PUBKEY_bio, i2d_DSA_PUBKEY_fp and
+i2d_DSAPublicKey
+
+See L</Deprecated low-level key reading and writing functions>
+and L<d2i_RSAPrivateKey(3)/Migration> 
+
+=item i2d_ECParameters, i2d_ECPrivateKey, i2d_ECPrivateKey_bio,
+i2d_ECPrivateKey_fp, i2d_EC_PUBKEY, i2d_EC_PUBKEY_bio, i2d_EC_PUBKEY_fp and
+i2o_ECPublicKey.
+
+See L</Deprecated low-level key reading and writing functions>
+and L<d2i_RSAPrivateKey(3)/Migration> 
+
+=item i2d_RSAPrivateKey, i2d_RSAPrivateKey_bio, i2d_RSAPrivateKey_fp.
+i2d_RSA_PUBKEY, i2d_RSA_PUBKEY_bio, i2d_RSA_PUBKEY_fp, i2d_RSAPublicKey
+i2d_RSAPublicKey_bio and i2d_RSAPublicKey_fp
+
+See L</Deprecated low-level key reading and writing functions>
+and L<d2i_RSAPrivateKey(3)/Migration> 
+
+=item IDEA_encrypt, IDEA_set_decrypt_key, IDEA_set_encrypt_key,
+IDEA_cbc_encrypt, IDEA_cfb64_encrypt, IDEA_ecb_encrypt and IDEA_ofb64_encrypt
+
+See L</Deprecated low-level encryption functions>.
+IDEA has been moved to the L<Legacy Provider|/Legacy Algorithms>.
+
+=item MD2, MD2_Init, MD2_Update and MD2_Final
+
+See L</Deprecated low-level encryption functions>.
+MD2 has been moved to the L<Legacy Provider|/Legacy Algorithms>.
+
+=item MD4, MD4_Init, MD4_Update, MD4_Final and MD4_Transform
+
+See L</Deprecated low-level encryption functions>.
+MD4 has been moved to the L<Legacy Provider|/Legacy Algorithms>.
+
+=item MDC2, MDC2_Init, MDC2_Update and MDC2_Final
+
+See L</Deprecated low-level encryption functions>.
+MDC2 has been moved to the L<Legacy Provider|/Legacy Algorithms>.
+
+=item IDEA_options and MD2_options
+
+These functions return constant strings and are no longer supported.
```

Comment:
Would it make sense to split this into two sections.  One for IDEA_options and one for MD2_options?  That way the IDEA_options could be positioned after the IDEA_* stuff above.